client/components/mma/cancel/cancellationContributionReminder.tsx

```diff
@@ -38,7 +38,7 @@ interface ReminderChoice {
 	thankYouMessage: string;
 }
 
-const REMINDER_ENDPOINT = '/api/reminders';
+const REMINDER_ENDPOINT = '/api/reminders/create';
```

Comment:
for clarity I've renamed this endpoint for the existing "winback" reminder signup. We now have `/create` and `/create-public`

Reply:
This existing one requires a user to be signed in